lib/transforms/analyzer.cc

```diff
@@ -257,7 +266,8 @@ static void RunOnInstruction(MatMulInst* inst,
                           ? matrixa_type.GetNumOfElementsInDim(dims - 1)
                           : matrixa_type.GetNumOfElementsInDim(dims - 2);
 
-  node_info.activation = node_info.sizeof_dt * (node_info.weight + inst->GetResultType().GetTotalNumOfElements());
+  node_info.activation = static_cast<float>(Dl->Bytes(weight_type) +
```

Comment:
are all the "static_cast<float>" needed?

Reply:
I think so, otherwise I see compiler warnings.